app/src/main/java/org/simple/clinic/main/TheActivityComponent.kt

```diff
@@ -137,7 +138,8 @@ interface TheActivityComponent :
     FacilityChangeScreen.Injector,
     ConfirmFacilityChangeSheet.Injector,
     ContactPatientBottomSheet.Injector,
-    SetAppointmentReminderView.Injector {
+    SetAppointmentReminderView.Injector,
+    ScheduleAppointmentSheet.Injector {
```

Comment:
Because we are no longer using a separate activity component for the sheet. Since this is a fragment inside the `TheActivity` we can use the `TheActivityComponent`